osbs/build/build_request.py

```diff
@@ -1112,6 +1114,11 @@ def render(self, validate=True):
             self.template['metadata'].setdefault('labels', {})
             self.template['metadata']['labels']['koji-task-id'] = str(koji_task_id)
 
+        filesystem_koji_task_id = self.spec.filesystem_koji_task_id.value
+        if filesystem_koji_task_id is not None:
+            self.template['metadata']['labels']['filesystem-koji-task-id'] = \
```

Comment:
Let's use the helper method `set_label` in this class.
FTW let's do the same for the `koji-task-id` above :)